fiat-file/src/main/java/com/netflix/spinnaker/fiat/roles/file/FileBasedUserRolesProvider.java

```diff
@@ -102,7 +102,7 @@
     List<Role> roles;
 
     public List<Role> getRoles() {
-      return roles.stream()
+      return roles == null ? Collections.emptyList() : roles.stream()
```

Comment:
Simple style nit - I think removing the turnary is easier to understand:

```
if (roles == null) {
  return Collections.emptyList();
}
return roles.stream()...
```

Reply:
Done.